spec/reek/report/location_formatter_spec.rb

```diff
@@ -0,0 +1,37 @@
+require_relative '../../spec_helper'
+require_relative '../../../lib/reek/report/formatter'
+require_relative '../../../lib/reek/smells/duplicate_method_call'
+require_relative '../../../lib/reek/smells/smell_warning'
+
+RSpec.describe 'location formatters' do
+  let(:warning) do
+    Reek::Smells::SmellWarning.new(Reek::Smells::DuplicateMethodCall.new,
+                                   lines:   [11, 9],
+                                   message: 'Foo#bar calls baz.qux 2 times',
+                                   source:  'string')
```

Comment:
Good point!

I never use FactoryGirl in my projects as I believe an elaborate test setup means there’s a missing factory method in the production code – which (a) would be then tested and kept working and (b) would also be helpful in irb sessions – but as we have FactoryGirl already I’ll switch to it later today. :)